codice/ddf
# DDF-05019 Balance UI font sizes

What are the relevant tickets?
Fixes: #5019

#### What does this PR do?
This PR Aims to start balancing font sizes in the UI, including some areas where FontAwesome icons are used.  

#### Who is reviewing it? 
@maryformanek 
@lifenouveau 
@djblue 

#### Ask 2 committers to review/merge the PR and tag them here.
@adimka 
@andrewkfiedler 

#### How should this be tested?
1. Open DDF
2. Check that the font sizes have been updated to reflect the medium font size. This PR mostly affects text in the main navigation bar at the top, but a few other key areas have the update as well. 

#### Any background context you want to provide?
This PR is part of a larger project that aims to improve the overall typography throughout the UI by balancing the body font sizes and and establishing proper hierarchies. 

#### Screenshots
![image](https://user-images.githubusercontent.com/11765980/61495967-8fc49580-a96f-11e9-845e-b343090676d6.png)
![image](https://user-images.githubusercontent.com/11765980/61496001-b256ae80-a96f-11e9-8439-b06ec066cd2c.png)

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
